source/dub/dub.d

```diff
@@ -864,7 +864,7 @@ class Dub {
 		if (options & FetchOptions.printOnly) {
 			if (existing && existing.version_ != ver)
 				logInfo("A new version for %s is available (%s -> %s). Run \"%s\" to switch.",
-					packageId.color(Mode.bold), existing.version_, ver,
+					packageId.color(Mode.bold), existing, ver,
```

Comment:
version_ crashes with non-version-range dependency specifications, so for logging we just dump the Dependency directly